package.json

```diff
@@ -40,7 +40,7 @@
     "eslint": "3.18.0",
     "eslint-config-xo": "0.18.1",
     "eslint-plugin-security": "^1.3.0",
-    "gulp": "github:gulpjs/gulp#4.0",
+    "gulp": "github:gulpjs/gulp#3f5aba2",
```

Comment:
Which means we don't get new free stuff (features, bugfixes). I guess this is the best way for now, since semver cannot be applied to branches.